icinga2/resource_icinga2_host_test.go

```diff
@@ -16,6 +16,7 @@ func TestAccCreateBasicHost(t *testing.T) {
 		hostname      = "terraform-host-1"
 		address       = "10.10.10.1"
 		check_command = "hostalive"
+		groups = ["linux-servers"]
```

Comment:
I would remove this from the test for creating a basic host and create a new test for testing/verifying groups.